class-tgm-plugin-activation.php

```diff
@@ -2713,7 +2713,7 @@ protected function get_row_actions( $item ) {
 			}
 
 			$prefix = ( defined( 'WP_NETWORK_ADMIN' ) && WP_NETWORK_ADMIN ) ? 'network_admin_' : '';
-			return apply_filters( "tgmpa_{$prefix}plugin_action_links", array_filter( $action_links ), $item['slug'], $item, $this->view_context );
+			return apply_filters( 'tgmpa_' . $prefix . 'plugin_action_links', array_filter( $action_links ), $item['slug'], $item, $this->view_context );
```

Comment:
AFAIK, filters (and the action hook below) need to be named with interpolation, for WP-Parser to be able to read the single string. Unless WP-Parser has been updated to read the first arg token, instead of the first string token...